.babelrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  presets: [
+    ["@babel/env", {
+      modules: process.env.NODE_ENV.includes('cjs') ||
+      process.env.NODE_ENV === 'test'
+      ? 'commonjs'
+      : false
+    }],
+    "@babel/react"
+  ],
+  plugins: []
+}
+
+if (process.env.NODE_ENV.includes('emotion')) {
```

Comment:
This could still be handled with the `env` field right? Sorry haven't had time to dig into some of the Babel 7 changes yet

Reply:
It's not deprecated (well, it was for a while).

`env` field just not used by js config. Consider json version with `env` field is a wrapper around it. Don't know where this mentioned in docs to be honest.